examples/serial_relay/serial_relay.ino

```diff
@@ -1,15 +1,15 @@
 #include <A6lib.h>
 
 // Instantiate the library with TxPin, RxPin.
-A6lib A6l(D6, D5);
```

Comment:
I tried working on it for 3 different versions of arduino IDE 1.6.8, 1.6.9 and 1.6.5 Linux.  Didn't work for me on any of these.
also not working on platformio with version 3.1.0 on linux.